### ISSUE TITLE
BEX - User Guides Fees

### ISSUE THREAD

**Issue by larrythecucumber321** [berachain/docs-old#376]
**Context:**
Review this section to make sure that all wording matches the latest information.

**Page:**
https://berachain-docs-bex-git-feat-bex-v2-berachain1.vercel.app/learn/guides/fees

**Tasks:**
- Complete rewrite for Balancer-BEX

**Comment by blackbera** [comment]
@DonBera - Fees
https://berachain-docs-bex-next.vercel.app/learn/guides/fees

**Comment by blackbera** [comment]
PR - https://github.com/berachain/docs/pull/440

**Comment by DonBera** [comment]
can't comment on the link. 

BTW I would add reference that fee starts from 0.01% not 0.3% (0.3% is min for weighted but stable have lower fees). 
Last but not least I think that fee for BGT staker is 50% the value of fee chosen in the pool not a fix 0.1%. 

Thanks!

**Comment by blackbera** [comment]
Preview Link - https://berachain-docs-bex-next.vercel.app/learn/guides/fees
Issue - https://github.com/berachain/docs/pull/497

cc @sj448

**Comment by sj448** [comment]
> Preview Link - https://berachain-docs-bex-next.vercel.app/learn/guides/fees Issue - #497
> 
> cc @sj448

These are the fees limit which are defined on smart contract level (same for both type of pools)
    uint256 private constant _MIN_SWAP_FEE_PERCENTAGE = 1e12; // 0.0001%
    uint256 private constant _MAX_SWAP_FEE_PERCENTAGE = 1e17; // 10%
@DonBera i think abv limit you mentioned are controlled on the UI so if the purpose of the doc is to help kinda educate on the UI side, looks fine to me

**Comment by sj448** [comment]
also not always 50% of swap fees moves to BGT stakers, we have the functionality to route some fee out of that 50% to `feeReceiver` (think of it as treasury addr) that being said initially whole 50% will move to polFeeCollector, see if we wanna add this

**Comment by DonBera** [comment]
> > Preview Link - https://berachain-docs-bex-next.vercel.app/learn/guides/fees Issue - [#497](https://github.com/berachain/docs/pull/497)
> > cc [@sj448](https://github.com/sj448)
> 
> These are the fees limit which are defined on smart contract level (same for both type of pools) uint256 private constant _MIN_SWAP_FEE_PERCENTAGE = 1e12; // 0.0001% uint256 private constant _MAX_SWAP_FEE_PERCENTAGE = 1e17; // 10% [@DonBera](https://github.com/DonBera) i think abv limit you mentioned are controlled on the UI so if the purpose of the doc is to help kinda educate on the UI side, looks fine to me

yes, I was referring to UI related action and financial meaning. 
A stable pool being super efficient can allow very small Fees while Weighted pool better to have at least 0.3% fee. 

On fee split worth mentioning 50% stays to LP and 50% goes to PoL ( or to BGT stakers or to foundation)? I think it can also remain an internal detail @blackbera what do you think?

**Comment by blackbera** [comment]
waiting for one last pass through with @sj448 on this

**Comment by sj448** [comment]
> https://berachain-docs-bex-next.vercel.app/learn/guides/fees

Do we want to add below detail?
Fee split is 50/50 between LPs and protocol.
Fees collected by protocol{inside ProtocolFeesCollector} is then split between BGT staker and treasury, ratio of which is configurable {polFeeCollectorPercentage inside ProtocolFeesWithdrawer contract} but most of the portion will go to BGT staker.

**Comment by sj448** [comment]
<img width="740" alt="Image" src="https://github.com/user-attachments/assets/f0998cb2-30a6-4ff9-9986-75924d1cb864" />

should replace BGT staker in this image with protocol, as in later section protocol fee split is explained.

rest all technically looks good, @DonBera please check from ur end too

**Comment by DonBera** [comment]
fixed moving to ready
